components/services/getAddress/index.js

```diff
@@ -0,0 +1,59 @@
+// Axios
+import axios from 'axios'
+
+// React
+import { func, number, string } from 'prop-types'
+
+const API_KEY = 'AG2YtZS2HEKCTOsZcDCFTg28696'
```

Comment:
I have two suggestion here:
1- Get the API_KEY as a prop, and have this one as default prop.
2- Get the API_KEY from `Config` context and have this one as default there.
I personally prefer the second one but you also reflect your opinion on this.

Reply:
the second one sounds better, please let me know about these improvements since you are more familiar with IUI